ompi/mca/mtl/psm2/mtl_psm2_component.c

```diff
@@ -334,6 +312,10 @@ ompi_mtl_psm2_component_query(mca_base_module_t **module, int *priority)
 static int
 ompi_mtl_psm2_component_close(void)
 {
+#if OPAL_CUDA_SUPPORT
+    if (cuda_envvar_set)
+        opal_unsetenv("PSM2_CUDA", &environ);
```

Comment:
Minor nit: please use `{}`, even for 1-line blocks.

Reply:
Fixed this